src/quo/components/info/information_box/view.cljs

```diff
@@ -26,7 +26,7 @@
      :container-style style/close-button}]])
 
 (defn- content
-  [{:keys [theme type button-label on-button-press message]}]
+  [{:keys [theme type button-label on-button-press message button-icon-right]}]
```

Comment:
We needed an icon for the info-box's button (`:i/external`), here it adds the parameter